### PR TITLE
Silo library from LLNL

### PR DESCRIPTION
**Pull request by datafl4sh**

### CODE REVIEW

File: silo.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  depends_on :x11
+  depends_on :fortran
+  depends_on "readline"
+
+  option "with-x",       "Use the X Window System"
+
+  def install
+    args = ["--prefix=#{prefix}"]
+    args << "--enable-optimization"
+    args << "--enable-x"        if build.with? "x"
```
Comment (dpo) [inline silo.rb]: `build.with? "x11"`

File: silo.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    args << "--enable-silex"    if build.with? "silex"
```
Comment (dpo) [inline silo.rb]: There's no option or dependency named `silex`.

File: silo.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    ENV.append "LDFLAGS", "-L#{Formula["readline"].opt_lib} -lreadline"
+    system "./configure", *args
+    system "make", "install"
+    rm "#{lib}/libsilo.settings"
```
Comment (dpo) [inline silo.rb]: `lib/"libsilo.settings"`

File: silo.rb [hunk 4]
```diff
@@ -0,0 +1,43 @@
+class Silo < Formula
+  desc "LLNL Silo library"
```
Comment (dpo) [inline silo.rb]: Could you indicate what the library does in a few words?

File: silo.rb [hunk 5]
```diff
@@ -0,0 +1,43 @@
+class Silo < Formula
+  desc "LLNL Silo library. Allows to create databases for VisIt."
+  homepage "https://wci.llnl.gov/simulation/computer-codes/silo"
+  url "https://wci.llnl.gov/content/assets/docs/simulation/computer-codes/silo/silo-4.10.2/silo-4.10.2-bsd.tar.gz"
+  sha256 "4b901dfc1eb4656e83419a6fde15a2f6c6a31df84edfad7f1dc296e01b20140e"
+
+  option "with-x11",        "Use the X Window System"
```
Comment (dpo) [inline silo.rb]: Sorry if that wasn't very clear earlier. Please remove this option and change line 9 to
```
depends_on :x11 => :optional
```

**Comment by dpo** [comment]
Thank you! I fixed the audit warnings.

**Comment by datafl4sh** [comment]
I made a little mess with this formula, thank you to you for your help!

**Pull request closed without merging**